### PR TITLE
oneAPI covariance distr oneCCL sample

### PR DESCRIPTION
**Pull request by mchernov-intel**

### CODE REVIEW

File: samples/daal/cpp/oneccl/sources/error_handling.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+/*
+!  Content:
+!    Auxiliary error-handling functions used in C++ samples
+!******************************************************************************/
+
+#ifndef _ERROR_HANDLING_H
+#define _ERROR_HANDLING_H
+
+const int fileError = -1001;
```
Comment (KulikovNikita) [inline samples/daal/cpp/oneccl/sources/error_handling.h:26]: Can it be obtained from any enum or smth like that?

**Review by KulikovNikita — APPROVED** [review]

File: samples/daal/cpp/oneccl/sources/error_handling.h (hunk 1 above)
Comment (amgrigoriev) [reply samples/daal/cpp/oneccl/sources/error_handling.h:26]: The file exists in master. Please update your fork

Comment (mchernov-intel) [reply samples/daal/cpp/oneccl/sources/error_handling.h]: rebased

**Review by amgrigoriev — APPROVED** [review]

**Pull request merged**